src/translations/translations.en.js

```diff
@@ -456,6 +462,13 @@ export const translations = {
   'cc-jenkins-info.update.title': `Updates`,
   'cc-jenkins-info.update.up-to-date': `Your Jenkins version is up-to-date.`,
   //#endregion
+  //#region cc-logs
+  'cc-logs.copied.multi': ({ count }) => `Copied (${count} lines)`,
```

Comment:
**suggestion:** I think we should use the same translation for 1 or multi lines and rely on our `pluralize` helper.

For 1 line, as a user, I'd rather get a message `Copied (1 line)` than `Copied`.